### PR TITLE
OCM-9353 | feat: update Version and MachineType

### PR DESCRIPTION
**Pull request by lucasponce**
Update the MachineType.architecture and Version.ReleaseImages

### CODE REVIEW

File: model/clusters_mgmt/v1/release_image_details_type.model [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+struct ReleaseImageDetails {
+    // ReleaseImage contains the URI of Openshift release image.
+    ReleaseImage String
+
+	// AvailableUpgrades is the list of versions this version can be upgraded to.
+	AvailableUpgrades []String
```
Comment (ciaranRoche) [inline model/clusters_mgmt/v1/release_image_details_type.model:22]: NP : indentation here needs to be fixed

File: model/clusters_mgmt/v1/machine_type_type.model [hunk 2]
```diff
@@ -42,4 +42,7 @@ class MachineType {
 
 	// 'true' if the instance type is supported only for CCS clusters, 'false' otherwise.
 	CCSOnly Boolean
+
+    // The architecture of the machine type.
```
Comment (ciaranRoche) [inline model/clusters_mgmt/v1/machine_type_type.model:46]: NP : Indentation on this as well :+1:

**Review by ciaranRoche — COMMENTED** [review]
Just some nit picks on indentation on this is all

**Review by ciaranRoche — APPROVED** [review]
lgtm

**Comment by lucasponce** [comment]
Thanks @ciaranRoche, the model files escaped to the IDE indentantionm

**Pull request merged**